### ISSUE TITLE
About the result

### ISSUE THREAD

**Issue by Zhaozeyong** [WujiangXu/EPO#2]
Hello, your work is very outstanding, but as far as I know, the difficulty of ScienceWorld dataset is much higher than ALFworld, but why is the success rate of EPO in ScienceWorld much higher than ALFworld in your work, which makes me feel very confused

**Comment by Zhaozeyong** [comment]
What do succ.* and succ. Stand for, respectively, the success rate on the training set? Could you give me a detailed answer? Thank you very much

**Comment by WujiangXu** [comment]
Hi, on Sciworld we use the 7B model while on Alfworld we use the 3B model. For the metric, you could refer to the paper with the explanation of "Succ.∗ reports the average of maximum success rates across random seeds, while Succ. measures average performance after convergence, reflecting practical reliability." A better way to study the performance is to compare via a plot, such as a wandb report.
